src/Components/Common/Header/style.ts

```diff
@@ -1,12 +1,15 @@
 import styled from "styled-components";
 import device from "Shared/Config";
-
 interface NavBar {
   active: boolean;
 }
 
-export const Header = styled.header`
-  position: fixed;
+interface header {
+  visible: boolean;
+}
+
+export const Header = styled.header<header>`
+  position: ${(props) => (props.visible ? "absolute" : "fixed")};
```

Comment:
이건 그냥 팁인데
```suggestion
  position: ${({visible}) => (visible ? "absolute" : "fixed")};
```
위와 같이 분해할당으로 더욱 간단하게 적을수 있습니다

Reply:
감사합니다!!